util/gh-pages/index_template.html

```diff
@@ -0,0 +1,262 @@
+<!DOCTYPE html>
+<!--
+Welcome to a Clippy's lint list, at least the source code of it. If you are
+interested in contributing to this website checkout `util/gh-pages/index.html`
```

Comment:
This comment should be updated to point to the template file

Reply:
Good catch, thanks!